Readme.md

```diff
@@ -209,9 +207,9 @@ Your must implement the ```EventListenerInterface```:
             $this->systemLogger->log('--- ConsoleOutputListener say something has been tagged ---');
         }
     }
-    
-The event handler locator can throw exception is something wrong with your command handler definition, please check your 
-system log to have more informations.
+
+The event handler locator can throw exception is something wrong with your command handler definition, please check your
```

Comment:
Let's do one single big doc cleanup PR to fix all the small typos and grammar issues.